doc/migration_registration_api_clients.md

```diff
@@ -0,0 +1,44 @@
+# Migrating Registration API Clients
+
+The `registration` API has been deprecated and will be removed from a future
+release. The new API surface is a superset of the previous functionality and
+provides consistent semantics, batch, and paging support.
+
+This document outlines the replacement RPCs when migrating clients away from
+the old registration API.
+
+## Replacement RPCs
```

Comment:
The `ListFederatedBundles` RPC is missing here.

Reply:
Whoa! I must have accidentally deleted the line in my typing fury! Thanks for catching that.